modules/ROOT/pages/create-map-inbound-edifact-json-xml.adoc

```diff
@@ -11,7 +11,7 @@ To use the template to create the mapping:
 . Set the Target JSON or XML Message Type
 . Test the map locally.
 
-[[edifact-create-inound-project]]
+[[edifact-create-inbound-project]]
```

Comment:
Changing an anchor affects other articles that reference this anchor. Please ensure that you perform a search of the string **[[edifact-create-inound-project]]** in all this project and ensure that any references found in other files are fixed to point to this new anchor name